ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/onboarding/AndroidOnboarding.kt

```diff
@@ -47,8 +51,18 @@ internal fun WrapOnboarding(
 
     // TODO [#383]: https://github.com/zcash/secant-android-wallet/issues/383
```

Comment:
Not related to this PR but it helps to have a small description of the TODO mostly for plain text readers like Github PR review UI